Makefile

```diff
@@ -59,12 +59,17 @@ integration:
 
 check: unit integration
 
+curtin:
```

Comment:
Thanks for helping bring this stuff to the Makefile, I like where this is going.
Under this plan, curtin and probert are going to have different results if they need an update and have already been checked out.
I suggest making these calls idempotent.
Because probert does the directory check, it can get stuck on an old version, where curtin will refuse to clone when there is one already.

Reply:
I updated my script to be more idempotent, see what you think. It won't quite do the right thing when we are tracking a branch and the branch moves but we've been using full hashes for a while now and I don't want to perform network access to check if the branch has moved every time we do anything so this is ok for now?

Reply:
Thanks!